pipelines/norsk-mirror.yml

```diff
@@ -28,6 +41,32 @@ jobs:
   - task: check-status-file
     file: ci/tasks/check-norsk-status.yml
     input_mapping: {scan-status: latest-scan-status}
+  on_success: &fixed-status
```

Comment:
if we're not using these anchors, I think it might be better removing them 🤔  e.g., it's easy to think that the this object is the same as `check-p-concourse-4.2.x`'s on-success given that the anchors indicate as if they were the same (although we're not dereferencing this one over there)

wdyt?

thx!

Reply:
haha yes good catch!